test/integration/targets/win_file/tasks/main.yml

```diff
@@ -16,506 +15,1063 @@
-#    - result.mode == '0444'
-#
-#- name: set sticky bit with symbolic mode
-#  win_file: dest={{win_output_dir}}/test_symbolic state=touch mode=o+t
-#  register: result
-
-#- name: assert file mode
-#  assert:
-#    that:
-#    - result.mode == '01444'
-#
-#- name: remove sticky bit with symbolic mode
-#  win_file: dest={{win_output_dir}}/test_symbolic state=touch mode=o-t
-#  register: result
-#
-#- name: assert file mode
-#  assert:
-#    that:
-#    - result.mode == '0444'
-
-#- name: add setgid with symbolic mode
-#  win_file: dest={{win_output_dir}}/test_symbolic state=touch mode=g+s
-#  register: result
-#
-#- name: assert file mode
-#  assert:
-#    that:
-#    - result.mode == '02444'
-#
-#- name: remove setgid with symbolic mode
-#  win_file: dest={{win_output_dir}}/test_symbolic state=touch mode=g-s
-#  register: result
-#
-#- name: assert file mode
-#  assert:
-#    that:
-#    - result.mode == '0444'
-
-#- name: add setuid with symbolic mode
-#  win_file: dest={{win_output_dir}}/test_symbolic state=touch mode=u+s
-#  register: result
-#
-#- name: assert file mode
-#  assert:
-#    that:
-#    - result.mode == '04444'
-
-#- name: remove setuid with symbolic mode
-#  win_file: dest={{win_output_dir}}/test_symbolic state=touch mode=u-s
-#  register: result
-#
-#- name: assert file mode
-#  assert:
-#    that:
-#    - result.mode == '0444'
+- name: create symbolic link folder
+  win_file:
+    path: "{{win_output_dir}}\\symbolic-link"
+    src: "{{win_output_dir}}\\dir\\sub-dir"
+    state: link
+  register: create_symbolic_directory
 
-# test the file module using follow=yes, so that the target of a
-# symlink is modified, rather than the link itself
+- name: get stat on created symbolic link folder
+  win_stat:
+    path: "{{win_output_dir}}\\symbolic-link"
+  register: create_symbolic_directory_stat
 
-#- name: create a test file
-#  win_copy: dest={{win_output_dir}}\test_follow content="this is a test file\n" mode=0666
+- name: assert create symbolic link folder changed and was created
+  assert:
+    that:
+    - create_symbolic_directory|changed
+    - create_symbolic_directory_stat.stat.exists
+    - create_symbolic_directory_stat.stat.islnk
+    - create_symbolic_directory_stat.stat.lnk_source == win_output_dir + '\dir\sub-dir'
 
-#- name: create a symlink to the test file
-#  win_file: path={{win_output_dir}}\test_follow_link src="./test_follow" state=link
-#
-#- name: modify the permissions on the link using follow=yes
-#  win_file: path={{win_output_dir}}\test_follow_link mode=0644 follow=yes
-#  register: result
-
-#- name: assert that the chmod worked
-#  assert:
-#    that:
-#    - result.changed
-#
-#- name: stat the link target
-#  win_stat: path={{win_output_dir}}/test_follow
-#  register: result
-#
-#- name: assert that the link target was modified correctly
-#  assert:
-#    that:
-##    - result.stat.mode == '0644'
+- name: create symbolic link folder again
+  win_file:
+    path: "{{win_output_dir}}\\symbolic-link"
```

Comment:
Should be able to single-quote all these and avoid the double-backslash escaping